BraveRewardsUI/Settings/Auto-Contribute/Details/AutoContributeDetailsViewController.swift

```diff
@@ -318,7 +318,7 @@ extension AutoContributeDetailViewController: UITableViewDataSource, UITableView
       let cell = tableView.dequeueReusableCell(for: indexPath) as AutoContributeCell
       cell.selectionStyle = .none
       
-      if let url = state.faviconURL {
+      if let url = URL(string: publisher.url) {
```

Comment:
Shouldn't this be `faviconURL`?

Reply:
`faviconURL` has value `clear`(?) for publishers, idk if it's something we have to set